sedkit/sed.py

```diff
@@ -1331,13 +1337,15 @@ def find_Simbad(self, search_radius=None, include=['parallax', 'spectral_type',
             # Print info
             n_rec = len(viz_cat)
             self.message("{} record{} for {} found in Simbad.".format(n_rec, '' if n_rec == 1 else 's', crit))
+            viz_cat.pprint()
+            # print(viz_cat.colnames)  # see what Simbad columns are available
```

Comment:
Is the commented out print statement here leftover from when you were fixing it ? If so we should get rid of it.